armbian/base/build/customize-armbian-rockpro64.sh

```diff
@@ -293,7 +295,7 @@ git clone https://github.com/ElementsProject/lightning.git || true
 cd lightning
 git checkout v${LIGHTNING_VERSION}
 ./configure
-make
+make -j
```

Comment:
I'll remove this. This belongs in a separate pr and should include a proper argument for the number of processors available.